Mathlib/Tactic/Substs.lean

```diff
@@ -11,5 +11,8 @@ open Tactic
 
 syntax (name := substs) "substs" (colGt ppSpace ident)* : tactic
 
+/--
+Applies the `subst` tactic to all given hypotheses from left to right
+-/
```

Comment:
The comment goes on the `syntax` (that's the doc string that normally shows up when hovering in user code). Also `.` at the end of the sentence.